### ISSUE TITLE
.\install.ps1 version isn't working

### ISSUE THREAD

**Issue by andschwa** [dotnet/cli#2934]
## Steps to reproduce

```
./install.ps1
```

## Expected  behavior

Usable version is installed.

## Actual behavior

```
dotnet_install: Downloading https://dotnetcli.blob.core.windows.net/dotnet/beta/Binaries/1.0.0-preview1-002692/dotnet-dev-win-x64.1.0.0-preview1-002692.zip 
dotnet_install: Extracting zip from https://dotnetcli.blob.core.windows.net/dotnet/beta/Binaries/1.0.0-preview1-002692/dotnet-dev-win-x64.1.0.0-preview1-002692.zip 
dotnet_install: Adding to current process PATH: "C:\Users\appveyor\AppData\Local\Microsoft\dotnet\". Note: This change will not be visible if PowerShell was run as a child process. 
dotnet_install: Installation finished 

dotnet : Found dotnet SDK, but did not find dotnet.dll at [C:\Users\appveyor\AppData\Local\Microsoft\dotnet\sdk\1.0.0-preview1-preview1-002692\dotnet.dll]
At C:\projects\secret:139 char:33
+         Start-NativeExecution { dotnet restore $RestoreArguments }
+                                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (Found dotnet SD...692\dotnet.dll]:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
```

**Comment by piotrpMSFT** [comment]
@sridhar-ms it looks like version.txt is also broken by recent version change.

**Comment by Sridhar-MS** [comment]
Can you give the output of `./install.ps1 -Verbose`

cc @krwq

**Comment by Sridhar-MS** [comment]
The version.txt file has the right version in it - https://dotnetcli.blob.core.windows.net/dotnet/beta/dnvm/latest.win.x64.version

**Comment by Sridhar-MS** [comment]
This is side affect of this issue #2928 and MultiCoreJit feature.

**Comment by piotrpMSFT** [comment]
Fixed by #2941
